packages/editor-ui/src/hooks/cloud.ts

```diff
@@ -34,474 +10,4 @@ export const n8nCloudHooks: PartialDeep<ExternalHooks> = {
 			},
 		],
```

Comment:
keeping this here since we agreed to keep external hooks and deprecate in v2
https://n8nio.slack.com/archives/C03594NKD7W/p1719481324464849

Reply:
Maybe add a `@deprecated` notice and mention that these are being deprecated in v2